app/src/main/res/values/string-untranslated.xml

```diff
@@ -39,4 +39,19 @@
     <string name="daxTrackersBlockedCtaZeroText"><![CDATA[&#160;was trying to track you here. <br/><br/>I blocked it!<br/><br/> ☝️You can check the URL bar to see who is trying to track you when you visit a new site.️]]></string>
     <!-- !!! WAITING FOR CORRECTIONS FROM TRANSLATOR - DO NOT RESUBMIT TRANSLATIONS ABOVE !!! -->
 
+    <!-- Broken Sites-->
+    <string name="brokenSitesCategoriesTitle">Describe what happened:</string>
+    <string name="brokenSitesCategoriesHint">Describe what happened</string>
+    <string name="brokenSiteSubmitButton">Send Report</string>
+    <string name="brokenSiteDescription">Submitting an anonymous broken site report helps us debug these issues and improve the app.</string>
+
+    <string name="brokenSiteCategoryImages">Images didn\'t load</string>
+    <string name="brokenSiteCategoryPaywall">The site asked me to disable</string>
```

Comment:
is this the correct copy? site asked me to disable _what_?

Reply:
It is the correct copy, comes from the existing copy in the extensions and approved here https://app.asana.com/0/1125189844152671/1165569833751685